apps/wpcom-block-editor/src/common/reorder-block-categories.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/**
+ * External dependencies
+ */
+import { getCategories, setCategories } from '@wordpress/blocks';
+import domReady from '@wordpress/dom-ready';
+
+const categorySlugs = [ 'jetpack', 'coblocks', 'coblocks-galleries' ];
+
+domReady( function() {
+	//preserve order of other columns, and split matching
+	const { core: coreCategories, custom: unsorted } = getCategories().reduce(
+		( { core, custom }, category ) => {
+			const matchIndex = categorySlugs.indexOf( category.slug );
```

Comment:
Nit; what do you think about having `const match = categorySlugs.includes( category.slug )` instead since we're not interested in the specific index at this point?

Reply:
Just out of habit for better browser compatibility (includes is not supported by IE). It should be okay to use `includes` if we polyfill Array methods properly in this bundle.

Reply:
Maybe a clearer variable name would read better?
```jsx
const isCustomCategory = categorySlugs.indexOf( category.slug ) ; // or  const isCustomCategory = categorySlugs.includes( category.slug )
if ( isCustomCategory ) {
    return {
        core,
        custom: [ ...custom, category ],
	};
}
return {
    core: [ ...core, category ],
    custom,
};
```

Reply:
Valid points!

I like the clarity of `isCustomCategory` 👍 

>if we polyfill Array methods properly in this bundle.

Yup, if this ends up in core Gutenberg views, there's `wp-polyfill` covering it.